.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: CI implementation 
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3 
+    
+    - name: Install node required
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18.x
+
+    - name: Install Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.3.0'
+
+    - name: Install dependencies
```

Please give clear and concise name.
```suggestion
    - name: Install Ruby dependencies
```